external/webserver/http_client.c

```diff
@@ -476,6 +520,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 						http_dispatch_url(client, req);
 						process_finish = true;
 						read_finish = true;
+
+						len->sentence_start = sentence_end + 2;
```

Comment:
What does `2` mean? Please add the comment about that for better understanding.

Reply:
done